src/python/WMCore/Services/TagCollector/TagCollector.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, url=None, logger=None, **kwargs):
         responseType will be either xml or json
         """
         defaultURL = "https://cmssdt.cern.ch/SDT/cgi-bin/ReleasesXML"
+        self.cFileReleases = 'ReleasesXML'
```

Comment:
This will create a single cache file no matter the arguments provided by the client, so potentially returning wrong results.

Reply:
We should actually rely on `self.tcArgs` to come up with the cache file name. Just mind the data type ;)

Reply:
Ok, I just changed the name to include url path and callname. 
As for  `self.tcArgs`, this is already done when the full name is defined through the `Service.cacheFileName` method. It appends a hash, the verb (e.g.: GET) and arguments into it (besides this string I'm passing).
https://github.com/dmwm/WMCore/blob/45bc3fe29ee9c5bfeb36a46a3490458bdc80d4c4/src/python/WMCore/Services/Service.py#L187